web_listview_invert_selection/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+List Invert Selection
+====================
+
+Allow to invert the current selection of a list of records.
+
+Configuration
+=============
+
+No configuration is needed.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go on any selectable list view;
+#. Click on the "Invert Selection" button on the list header;
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/162/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
```

Comment:
Check the [updated template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst), this part of text is a bit changed.